packages/@o3r/core/ng-package.json

```diff
@@ -8,6 +8,6 @@
     "entryFile": "src/public_api.ts"
   },
   "allowedNonPeerDependencies": [
-    "chokidar", "comment-json", "globby", "semver", "tslib", "uuid"
+    "chokidar", "comment-json", "globby", "semver", "tslib", "uuid", "@o3r/schematics"
```

Comment:
`@o3r/schematics` not needed anymore

Reply:
same as prevoius comment, in o3r/core we keep it as direct dep.